app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -35,6 +35,9 @@ public enum Chain {
     Chain chaining = Chain.NOPE;
     int trapSucess = 0;
     int trapFail = 0;
+
+    double analScore = 0.0;
```

Comment:
Suggestion: use full name `analyzerScore`. Same for method name on line 148

Reply:
Done.